docs/tutorials.state_distinguishability.rst

```diff
@@ -204,33 +202,32 @@ distinguishability were initially considered in :cite:`Yu_2012_Four`. In :code:`
 we can calculate the probability with which Bob can distinguish these states
 via PPT measurements in the following manner.
 
-.. code-block:: python
-
-    >>> import numpy as np
-    >>> from toqito.states import bell
-    >>> from toqito.state_opt import ppt_distinguishability
-    >>> # Bell vectors:
-    >>> psi_0 = bell(0)
-    >>> psi_1 = bell(2)
-    >>> psi_2 = bell(3)
-    >>> psi_3 = bell(1)
-    >>>
-    >>> # YDY vectors from :cite:`Yu_2012_Four`:
-    >>> x_1 = np.kron(psi_0, psi_0)
-    >>> x_2 = np.kron(psi_1, psi_3)
-    >>> x_3 = np.kron(psi_2, psi_3)
-    >>> x_4 = np.kron(psi_3, psi_3)
-    >>>
-    >>> # YDY density matrices:
-    >>> rho_1 = x_1 @ x_1.conj().T
-    >>> rho_2 = x_2 @ x_2.conj().T
-    >>> rho_3 = x_3 @ x_3.conj().T
-    >>> rho_4 = x_4 @ x_4.conj().T
-    >>>
-    >>> states = [rho_1, rho_2, rho_3, rho_4]
-    >>> probs = [1 / 4, 1 / 4, 1 / 4, 1 / 4]
-    >>> np.around(ppt_distinguishability(vectors=states, probs=probs, dimensions=[2, 2, 2, 2], subsystems=[0, 2])[0], decimals=2)
-    np.float64(0.87)
+.. jupyter-execute::
+
+     import numpy as np
+     from toqito.states import bell
+     from toqito.state_opt import ppt_distinguishability
+     # Bell vectors:
+     psi_0 = bell(0)
+     psi_1 = bell(2)
+     psi_2 = bell(3)
+     psi_3 = bell(1)
+    
+     # YDY vectors from :cite:`Yu_2012_Four`:
+     x_1 = np.kron(psi_0, psi_0)
+     x_2 = np.kron(psi_1, psi_3)
+     x_3 = np.kron(psi_2, psi_3)
+     x_4 = np.kron(psi_3, psi_3)
+    
+     # YDY density matrices:
+     rho_1 = x_1 @ x_1.conj().T
+     rho_2 = x_2 @ x_2.conj().T
+     rho_3 = x_3 @ x_3.conj().T
+     rho_4 = x_4 @ x_4.conj().T
+    
+     states = [rho_1, rho_2, rho_3, rho_4]
```

Comment:
We could make this example a bit less verbose and not even have the need to construct the corresponding density matrices (the function itself should handle the case where just vectors are passed in):

```suggestion    
     states = [x_1, x_2, x_3, x_4]
```